src/view.c

```diff
@@ -1038,6 +1038,10 @@ view_set_untiled(struct view *view)
 	view->tiled = VIEW_EDGE_INVALID;
 	view->tiled_region = NULL;
 	zfree(view->tiled_region_evacuate);
+
+	if (view->impl->set_tiled) {
+		view->impl->set_tiled(view);
+	}
```

Comment:
I think we usually wrap these in a `view_set_tiled(view)` so we don't have to repeat the impl check everywhere.